app/controllers/proxy_controller.rb

```diff
@@ -1,17 +1,15 @@
 class ProxyController < ApplicationController
-  def configuration
-    @config ||= ::Configuration.new(okapi).tap do |config|
-      config.load!
-    end
-  end
+  before_action :slice_eholdings_namespace
 
   def index
-    # This is currently just a passthrough to the RM-API
-
-    # Form the URL
-    urlstr = "#{ENV['EBSCO_RESOURCE_MANAGEMENT_API_BASE_URL']}/rm/rmaccounts/#{configuration.customer_id}#{request.fullpath}"
-    urlstr.slice! '/eholdings'
-    uri = URI(urlstr)
+    # For the time being, this is a direct passthrough to
+    # the EBSCO RM-API.  As more of Codex is incorporated, this will
+    # be replaced with more robust serialization.
+    uri = URI(
+      "%s/rm/rmaccounts/%s%s" % [
```

Comment:
I'm not opposed to using this method of substitution instead of a template string, but if we go this route, let's use named keys instead of positional:

```ruby
"%{base}/rm/rmaccounts/%{customer}%{path}" % base: rmapi_base_url, customer: @customer_id, path: request.fullpath
```

Reply:
i didn't know you could use named keys in format strings!  that's excellent